scripts/oes-data-migration-scripts/migration_v3.12.x_to_v4.0.x.py

```diff
@@ -1437,7 +1449,7 @@ def postingGateJson(pipelineJson, cookie):
         response = requests.post(url=api_url, headers=headers, data=json.dumps(pipelineJson))
         logging.info("The response status is: " + str(response.status_code))
         if (response.status_code == 200 | response.status_code == 201):
-            logging.info("Successfully added stage! The response is:\n" + str(response.content) + '\n')
+            logging.info("Successfully added stage! The response is:\n" + str(response.text) + '\n')
         else:
             print("Failed to add stage; The response is: "+str(response.content))
             logging.info("Failed to add stage; The response is:\n" + str(response.content) + '\n')
```

Comment:
why don't we throw an exception and break the script execution ?
something like this - "raise Exception("Sapor API response shows failure!")"